var/spack/repos/builtin/packages/trimmomatic/package.py

```diff
@@ -43,7 +43,7 @@ class Trimmomatic(Package):
 
     def install(self, spec, prefix):
         mkdirp(prefix.bin)
-        jar_file = 'trimmomatic-{}.jar'.format(self.version.dotted)
+        jar_file = 'trimmomatic-{v}.jar'.format(v=self.version.dotted)
```

Comment:
Spack's going to teach my Python in spite of my best efforts.  Soon enough I'll be carrying the Python 2 and Python 3 variants of things around in my head.

:cold_sweat:....